packages/chat-headless/src/ChatHeadless.ts

```diff
@@ -31,12 +31,174 @@ import { getClientSdk } from "./utils/clientSdk";
 import { ChatClient } from "./models/ChatClient";
 
 /**
- * Provides the functionality for interacting with a Chat Bot
- * and the relevant application state.
+ * Provides the functionality needed to interact with the Chat API in a
+ * stateful manner.
  *
  * @public
  */
-export class ChatHeadless {
+export interface ChatHeadless {
```

Comment:
nit: move this interface into models/ folder as `ChatHeadless`, then rename this file as `ChatHeadlessIml`